terraso_backend/apps/auth/services.py

```diff
@@ -29,11 +29,12 @@
 from django.utils import timezone
 
 from apps.core.models import UserPreference
-from apps.core.models.users import NOTIFICATION_KEYS
+from apps.core.models.users import USER_PREFS_KEY_LANGUAGE, USER_PREFS_KEYS
 from apps.storage.services import ProfileImageService
 
 from .providers import AppleProvider, GoogleProvider, MicrosoftProvider
 from .signals import user_signup_signal
+from .utils import format_to_upper
```

Comment:
suggestion: what about using core_formatters instead of creating a new file? I would also give it a more specific name, like `uppercase_locale`.

Reply:
Ooh, didn't see that file existed, thanks!